cypress/support/api/utils/users.js

```diff
@@ -120,7 +120,7 @@ export function getMailWithResetPasswordLink(email, subject, i = 0) {
 export function getMailsForUser(email, i = 0) {
   const serverStoredEmail = email.toLowerCase();
 
-  if (i > 5) {
+  if (i > 9) {
```

Comment:
**question**: what is this number? 

Maybe it is worth extracting it to a new variable with the descriptive name?